scripts/variantstore/beta_docs/gvs-faq.md

```diff
@@ -0,0 +1,33 @@
+# Frequently Asked Questions about the Genomic Variant Store
+
+## Output questions
+1. Is the resulting callset different from the WARP Joint Calling Pipeline?
+   1. Yes, to enable scale, cost, and runtime efficiencies in GVS, we have optimized what data is output in the final VCFs. Please see details on the [outputs of gvs](./gvs-outputs.md).
+   2. Additionally, GVS defaults to use GATK VETS over the prior tool, GATK VQSR. See the [release notes](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/docs/release_notes/VETS_Release.pdf).
+2. How do I confirm the samples included in the callset?
+   1. Alongside the output vcfs there is a file called `sample-name-list.txt`
+3. How can I know which sharded vcf files should be merged together? Is there any file indicating the relationship between sharded vcf and chromosome?
+   1. Yes, please see details on the [outputs of gvs](./gvs-outputs.md).
+4. There are so many variants! I thought this was filtered.
+   1. The GVS 'soft filters' the data in that we tag site-level filters and allele specific filters onto the data for the user to be able to filter based on their research needs. Please see details on the [outputs of gvs](./gvs-outputs.md).
+5. Can you output to Hail VDS format?
+   1. Currently, in the GvsBeta, we can only support VCF formatted outputs. This is because Terra users cannot get the right access in workspaces to be able to run Hail in a WDL to be able to make the VDS. 
+6. Can you output to Plink pgen format?
+   1. We will release support to output to pgen in the future. Please contact variants@broadinstitute.org to let us know you are interested in this feature so we can prioritize it sooner!
+
+## Runtime
+1. For running GVS, do we need to make a separate BigQuery dataset for every callset we want to create, or just once and then that can hold multiple cohorts across GVS runs?
+   1. New data = new dataset in BQ. If adding data to an existing callset that hasn't been cleaned up, you could run GVS again in the same workspace and it would add the new data only and recreate the callset.
```

Comment:
I will clarify. Yea essentially saying that, but you're right that it's probably not worth mentioning that here since we don't have more documentation on how to use GVS in that way. I'll remove that part.